cmd/promu.go

```diff
@@ -108,7 +108,7 @@ func fatal(err error) {
 // fatalMsg prints a message and exit
 func fatalMsg(err error, msg string) {
 	if err != nil {
-		fmt.Println("!!", msg, err)
+		fmt.Println("!!", fmt.Sprintf("%s:", msg), err)
```

Comment:
Alternatively `fmt.Println("!!", msg, ":", err)`, if you prefer a format free one.